documentation/configure.markdown

```diff
@@ -200,17 +203,11 @@ PuppetDB can use either **a built-in HSQLDB database** or **a PostgreSQL databas
 >
 > Depending on demand, Oracle support may be forthcoming in a future version of PuppetDB. This hasn't been decided yet.
 
-### Using Built-in HSQLDB
-
-To use an HSQLDB database at the default `<vardir>/db`, you can simply remove all database settings. To configure the DB for a different location, put the following in the `[database]` section:
+### HSQLDB Deprecation {#hsqldb-deprecation}
```

Comment:
@wkalt could be.  I initially did that so I could refer to it from two places, but I imagine I could do the same if we moved it (just refer to the section it's in).